test-data/genops-generators.test

```diff
@@ -1809,25 +1809,25 @@ def recursive_outer_gen.__mypyc_generator_helper__(__mypyc_self__, type, value,
     r34 :: short_int
     r35, r36, r37 :: object
     r38, r39 :: bool
-    r40, r41 :: object
-    r42 :: int
-    r43 :: bool
-    r44 :: int
-    r45 :: object
-    r46 :: short_int
-    r47 :: bool
-    r48 :: object
-    r49, r50, r51 :: bool
-    r52 :: None
-    r53 :: object
-    r54 :: short_int
-    r55, r56 :: bool
-    r57 :: short_int
-    r58 :: bool
-    r59 :: short_int
-    r60 :: bool
-    r61 :: short_int
-    r62, r63 :: bool
+    r40, r41, r42 :: object
```

Comment:
They could be named and numbered separately by rtype as well? Would reduce diff noise

Reply:
Yeah that's not a bad idea.

(One secret is that there is an `--update-data` flag you can pass to pytest and mypyc's test harness will rewrite the expected output. This doesn't reduce noise but does reduce the effort.)

Honestly the genops testing stuff is just kind of bad. The real solution is to have something smart instead (#486), but absent that we should maybe just totally delete the generators ones in particular (#432).